### DISCUSSION TITLE
Feature Request: add chapter time to dropdown list

### DISCUSSION THREAD

**Discussion by jozic** [PaulWoitaschek/Voice#1424] [Ideas]
Right now when I click on chapters dropdown list it has only names, see screenshot
![screenshot_20171104-165157](https://user-images.githubusercontent.com/124249/32409460-77974006-c182-11e7-9c25-5e65db2d1f09.png)

Would be nice to have time of the chapter here as well.
Like 

`02 - Chapter 2 (12:36)`

If you don't mind I can try to implement it. If you don't like it, just close this issue.

Thanks

**Comment by jozic** [comment]
Hey @PaulWoitaschek wdyt? Should I try or should I give up?

**Comment by jozic** [comment]
I've given it a try and it looks like that
![image](https://user-images.githubusercontent.com/124249/32760977-d5c5d67c-c8be-11e7-9df7-80a050fec4e8.png)

I wouldn't mind if time would be at the very end of the lines, but not sure how to do that

Please let me know, if i should proceed with PR. If you don't like it, that's fine :)

**Comment by hyiltiz** [comment]
Mine (Google X5; can't find out the version number, would be great to say the version number, authors and license in the software in an About page) doesn't even show chapter list although it had chapter markers (iTunes and some other players can use the chapter checkpoints).

**Comment by hyiltiz** [comment]
Voice 3.6.3 recognizes an audiobook split into 4 m4b files as one audiobook, but detects those 4 files as separate chapters and lists those 4 files in the dropdown menu above, not the chapter markers in any of those files. Is that expected behavior? Can Voice only handle a single file audiobook as of now?
